.github/workflows/build_bundle_package.yml

```diff
@@ -94,9 +97,10 @@ jobs:
           export PATH=${PATH}:${MAVEN_HOME}/bin && \
           mvn clean install -P${{ github.event.inputs.spark }} -Dhadoop.version=${{ github.event.inputs.hadoop }} -Pbackends-velox -Pceleborn -Puniffle -DskipTests -Dmaven.source.skip
       - name: Upload bundle package
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
```

Comment:
may also need to fix the `download-artifact`

Reply:
yes